bootstrapping_tools/bootstrapping.py

```diff
@@ -6,12 +6,34 @@
 
 
 def power_law(time_interval, population_growth_rate, initial_population_size):
+    """This function represent a simple power lay of the type: f(y) = a*x^y
```

Comment:
¿Será _law_ en lugar de _lay_?
```suggestion
    """This function represent a simple power law of the type: f(y) = a*x^y
```